app/src/main/java/com/appunite/loudius/network/utils/ApiRequester.kt

```diff
@@ -44,11 +44,11 @@ class ApiRequester @Inject constructor(private val gson: Gson) {
     }
 
     private fun getApiErrorMessageIfExist(throwable: HttpException) = try {
-        val errorResponse = gson.fromJson(
```

Comment:
info: I fixed this code, because we had crashes in the app when API returned an error that didn't have those information provided.